shoop/themes/classic_gray/plugins/social_media_links.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from shoop.social_media.models import SocialMediaLink
+from shoop.xtheme import TemplatedPlugin
+from shoop.xtheme.plugins.forms import TranslatableField
+
+
+class SocialMediaLinksPlugin(TemplatedPlugin):
+    identifier = "classic_gray.social_media_links"
+    name = _("Social Media Links")
+    template_name = "classic_gray/plugins/social_media_links.jinja"
+    fields = [
+        ("topic", TranslatableField(label=_("Topic"), required=False, initial="")),
+        ("text", TranslatableField(label=_("Text"), required=False, initial="")),
+        ("icon_size", forms.ChoiceField(label=_("Icon Size"), required=False, choices=[
+            ("", "Default"),
+            ("lg", "Large"),
+            ("2x", "2x"),
+            ("3x", "3x"),
+            ("4x", "4x"),
+            ("5x", "5x"),
+        ], initial="")),
+        ("alignment", forms.ChoiceField(label=_("Alignment"), required=False, choices=[
+            ("", "Default"),
+            ("left", "Left"),
+            ("center", "Center"),
+            ("right", "Right"),
+        ], initial="")),
+    ]
```

Comment:
I think you should have ModelMultipleChoiceField here. Other solution would be to add enabled attribute to ``SocialMediaLink``s. Point is that we don't neccessary want to show every single link we have stored in database.